database/init_db.go

```diff
@@ -50,7 +50,7 @@ func main() {
 
 	// SQL instruction to create a table.
 	sqlStatement := `
-	CREATE TABLE IF NOT EXISTS public.test_objects (
+	CREATE TABLE IF NOT EXISTS public.archive (
```

Comment:
https://github.com/kubearchive/kubearchive/pull/134/

```suggestion
	CREATE TABLE IF NOT EXISTS public.resource (
```

Reply:
Fixed when rebasing